src/packages/README.md

```diff
@@ -9,17 +9,90 @@ They have different levels of completeness, maturity, security, and support.
 release, which differs substantially from prior releases and software
 snapshots due to the introduction of a `pkg64c` package set.**
 
+## Overview
+
+The following table presents an overview of available package managers in
+CheriBSD that are described in more details in consecutive sections.
+
+| ABI        | #      | Manager | Install path   | Suitable for     | Examples |
+|------------|--------|---------|----------------|------------------|----------|
+| Hybrid ABI | 20,000 | `pkg64` | `/usr/local64` | Day-to-day use   | bash<br/>gdb-cheri<br/>llvm-base<br/>vim |
+| CheriABI   | 8,000  | `pkg64c`| `/usr/local`   | Experimental use | dash<br/>git<br/>nano<br/>rsync |
+
+## Package managers in CheriBSD
+
+*Note:* As of this writing we only provide packages for Morello systems.
+We aim to add CHERI-RISC-V package sets in the near future.
+
+CheriBSD includes two package managers: `pkg64` for hybrid ABI packages and
+`pkg64c` for CheriABI packages. The FreeBSD package manager `pkg` is not
+available on CheriBSD. The `pkg` program was replaced with a script that prints
+information on available package managers and always returns the exit code 1
+making `pkg` commands invalid on CheriBSD:
+```
+# pkg
+Please use the ABI-specific pkg variant:
+* pkg64c for experimental software built for CheriABI;
+* pkg64 for stable software built for the hybrid ABI.
+
+Replace pkg with pkg64c or pkg64 in your command and try again.
+# echo $?
+1
+```
+
+We expect that `pkg64c` will be renamed to `pkg` in a future CheriBSD release.
+The intention is that, over time, the CheriABI packages will become more
+mature, and hence the preferred collection for day-to-day use.
+
+The syntax of the `pkg64` and `pkg64c` commands match the syntax of the `pkg`
+command from FreeBSD.
+You can find information on package manager commands in FreeBSD's `pkg` manual
+pages shipped with CheriBSD, e.g. for the commands `pkg64 rquery` and
+`pkg64c rquery`, execute `man pkg-rquery`.
+
+Additionally, you can learn more about FreeBSD's package manager by reading the
+[pkg(8) manual page](https://www.freebsd.org/cgi/man.cgi?pkg(7)) online
+and the [FreeBSD Handbook chapter on packages and
+ports](https://docs.freebsd.org/en/books/handbook/ports/).
+
 ## Hybrid ABI packages
 
 **Hybrid ABI packages** are compiled almost identically to packages in the
 baseline non-CHERI architecture (e.g., Armv8-a for Morello, and 64-bit
 RISC-V for RISC-V), and do not have improvements in memory protection or
 software compartmentalization.
```

Comment:
```suggestion
software compartmentalization. These packages are intended to provide stable versions of tools necessary to develop software and otherwise use your CHERI system while more software is ported to CheriABI.
```

Reply:
I applied your suggestion but in the second paragraph.